app-modules/notifications/src/Models/Subscription.php

```diff
@@ -40,6 +45,35 @@ public function subscribable(): MorphTo
         return $this->morphTo();
     }
 
+    public static function executeFromCampaignAction(CampaignAction $action): bool|string
+    {
+        try {
+            DB::beginTransaction();
+
+            $action->campaign->caseload->retrieveRecords()->each(function (Subscribable $subscribable) use ($action) {
+                if ($action->data['remove_prior']) {
+                    $subscribable->subscriptions()->delete();
+                }
```

Comment:
I asked a question to Product in this ticket: https://engage.canyongbs.com/workgroups/group/3/tasks/task/view/893/?IFRAME=Y&IFRAME_TYPE=SIDE_SLIDER&MID=3349#com3349

I think it may be that this setting is supposed to clear the Subscriptions of the selected Users, not the Subscribables in the Caseload.

Reply:
Product has confirmed it is indeed supposed to clear the Subscribers of the population.